frontend/src/components/Buttons/PlayButton.vue

```diff
@@ -35,92 +35,62 @@
   </div>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
```

Comment:
Github absolutely destroyed this diff, but the changes are actually very minimal. None of the logic was changed, just switched to composition setup api